components/catalogs/front/src/components/CatalogEdit/CatalogEdit.integration.tsx

```diff
@@ -97,6 +97,26 @@ test('it can add a product value filter on the channel', async () => {
             url: '/rest/catalogs/channels?page=1&limit=20',
             json: [ECOMMERCE, PRINT],
         },
+        {
+            url: '/rest/catalogs/channels?codes=',
```

Comment:
```suggestion
            url: '/rest/catalogs/channels?codes=',
```

Your PR is good and it's not related to what you did but damn, we should improve the way we call the API to stop passing empty parameters.

Reply:
Agreed : we should use [URLSearchParams](https://github.com/akeneo/pim-community-dev/blob/755a68e4878de5dcbbc272169b72cd18c3d0332e/components/catalogs/front/src/components/ProductSelection/hooks/useInfiniteAttributeOptions.ts#L45) as we did in some queries